src/canvas_assets/src/Submit.tsx

```diff
@@ -107,7 +107,7 @@ function Submit(props: { handleDrop: any; renderCanvas2: any }) {
             cancelLabel="cancel"
             isPrimaryActionDisabled={!isAuthenticated}
             width="size-500"
-            isHidden={!isConfirmDisabled && isAuthenticated}
+            isHidden={!isConfirmDisabled && !isAuthenticated}
```

Comment:
@krpeacock submit window was not opening for me. I think this should be correct. please double-check :)